Description: Fix FTBFS on kfreebsd
Author: Laurent Bigonville <bigon@debian.org>
Forwarded: https://github.com/Haivision/srt/issues/2253#issue-1154523220

Index: libsrt/srtcore/utilities.h
===================================================================
--- libsrt.orig/srtcore/utilities.h
+++ libsrt/srtcore/utilities.h
@@ -115,7 +115,7 @@ written by
 
 #	include <sys/endian.h>
 
-#elif defined(__NetBSD__) || defined(__FreeBSD__) || defined(__DragonFly__)
+#elif defined(__NetBSD__) || defined(__FreeBSD__) || defined(__DragonFly__) || defined(__FreeBSD_kernel__)
 
 #	include <sys/endian.h>
 
