Description: Fix a typo
Author: Florian Ernst <florian@debian.org>
Forwarded: https://github.com/Haivision/srt/issues/2254#issue-1154523301

Index: libsrt/srtcore/core.cpp
===================================================================
--- libsrt.orig/srtcore/core.cpp
+++ libsrt/srtcore/core.cpp
@@ -9338,7 +9338,7 @@ bool srt::CUDT::overrideSndSeqNo(int32_t
     const int diff = CSeqNo(seq) - CSeqNo(m_iSndCurrSeqNo);
     if (diff < 0 || diff > CSeqNo::m_iSeqNoTH)
     {
-        LOGC(gslog.Error, log << CONID() << "IPE: Overridding with seq %" << seq << " DISCREPANCY against current %"
+        LOGC(gslog.Error, log << CONID() << "IPE: Overriding with seq %" << seq << " DISCREPANCY against current %"
                 << m_iSndCurrSeqNo << " and next sched %" << m_iSndNextSeqNo << " - diff=" << diff);
         return false;
     }
